configs/legacy/README.md

```diff
@@ -0,0 +1,46 @@
+# Legacy Configs in MMDetection V1.x
+
+Configs in this directory implements the legacy configs used by MMDetection V1.x and its model zoos.
+
+To help users convert their models from V1.x to MMDetection V2.0, we provide v1.x configs to inference the converted v1.x models.
+Due to the BC-breaking changes in MMDetection V2.0 from MMDetection V1.x, running inference with the same model weights in these two version will produce different results. The difference will cause within 1% AP absolute difference as can be found in the following table.
+
+## Usage
+
+To upgrade the model version, the users need to do the following steps.
```

Comment:
We may also give some instructions on how to convert models that are not in the model zoo, e.g., custom detectors.